src/StatusQ/Buttons/StatusIconButton.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+
+Rectangle {
+    id: statusIconButton
+
+    property color iconNormalColor: "transparent"
+    property color iconDisabledColor: "transparent"
+
+    property color backgroundNormalColor: "transparent"
+    property color backgroundHoverColor: "transparent"
+    property color backgroundDisabledColor: "transparent"
```

Comment:
I'm probably missing something here, but at a first glance, this looks off to me.
All of these colors are `transparent` by default, meaning that, if I put a `StatusIconButton` in my app, I won't see a thing, unless I explicitly assign colors.

`Theme` has been built to take care of that.

Any reason you've made it this way?

Reply:
Just one reason - no logic for colors :) i will fix it by default.